### PR TITLE
fix pypi rendering

### PR DESCRIPTION
**Pull request by ocefpaf**
@Conengmo this should fix the bad rendering of the [README on PyPI](https://pypi.org/project/folium/).

I'm planning a new release soon. Do you have any PRs you would like to get merged before that?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,9 +43,6 @@ def walk_subpkg(name):
     'folium.plugins'
 ]
 
-LICENSE = read('LICENSE.txt')
-long_description = '{}\n{}'.format(read('README.rst'), read('CHANGES.txt'))
```
Comment (Conengmo) [inline setup.py]: Could we still have the changelog following the readme?

**Comment by Conengmo** [comment]
> I'm planning a new release soon. Do you have any PRs you would like to get merged before that?

Good plan! I'll update the changelog now first. I don't have any PR's that we have to wait for. I'm excited about adding the geojson markers and popups and refactoring the geojson styling, but no need to wait for that.

**Comment by ocefpaf** [comment]
> I'm excited about adding the geojson markers and popups and refactoring the geojson styling, but no need to wait for that.

We can issue a new one as soon as those are merged. Just keep me posted if you need help reviewing/testing those.

File: setup.py (hunk 1 above)
Comment (ocefpaf) [reply setup.py]: I can re-add that. I thought it was too verbose and boring for PyPI but I don't have strong opinions about it.

Comment (Conengmo) [reply setup.py]: Hmm yeah maybe. If someone wants to read the changelog they know where to find it I guess.

Comment (ocefpaf) [reply setup.py]: The CHANGELOG is versioned so I guess that a link to the latest master version would make that easy to find and still keeping it simple. What do you think?

**Comment by ocefpaf** [comment]
> Let's bake a release!

I'll start the processes as soon as this one is merged.

File: setup.py (hunk 1 above)
Comment (Conengmo) [reply setup.py]: That seems good.

**Comment by ocefpaf** [comment]
Self-merging this one b/c it has simple README modifications.

**Pull request merged**